src/components/action-toolbar/action-toolbar.js

```diff
@@ -44,15 +44,23 @@ class ActionToolbar extends React.Component {
      * @property actions - each action is object with the action attributes
      * @type {Array}
      */
-    actions: PropTypes.object.isRequired,
+    actions: PropTypes.array,
```

Comment:
`actions` should be an `object`

Reply:
Can you also update the example on line 19 in this file to use an object to avoid confusion